pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/excel/renderer/ValueProvider.java

```diff
@@ -0,0 +1,145 @@
+package org.broadinstitute.dsm.model.elastic.export.excel.renderer;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.dsm.model.Filter;
+import org.broadinstitute.dsm.model.ParticipantColumn;
+import org.broadinstitute.dsm.model.elastic.sort.Alias;
+import org.broadinstitute.dsm.statics.ESObjectConstants;
+import org.broadinstitute.dsm.util.ElasticSearchUtil;
+import org.broadinstitute.dsm.util.proxy.jackson.ObjectMapperSingleton;
+
+public interface ValueProvider {
```

Comment:
Since refactoring is started for better design, I would love to see unit tests as well.